TargetRenamer/Trados.TargetRenamer/Trados.TargetRenamer/View/TargetRenamerSettingsView.xaml

```diff
@@ -3,9 +3,151 @@
+			 xmlns:enums="clr-namespace:Trados.TargetRenamer.Helpers"
              xmlns:local="clr-namespace:Trados.TargetRenamer.View"
+             xmlns:System="clr-namespace:System;assembly=mscorlib"
              mc:Ignorable="d" 
+             d:DataContext="{d:DesignInstance viewModel:TargetRenamerSettingsViewModel}"
              d:DesignHeight="450" d:DesignWidth="800">
-    <Grid>
-	</Grid>
+	<UserControl.Resources>
+		<ResourceDictionary>
+			<ResourceDictionary.MergedDictionaries>
+				<ResourceDictionary Source="/Sdl.Desktop.Platform.Styles;component/GroupboxResources.xaml" />
+				<ResourceDictionary Source="/Sdl.Desktop.Platform.Styles;component/Checkboxresources.xaml" />
+				<ResourceDictionary Source="/Sdl.Desktop.Platform.Styles;component/textboxresources.xaml" />
+				<ResourceDictionary Source="/Sdl.Desktop.Platform.Styles;component/comboboxresources.xaml" />
+				<ResourceDictionary Source="/Sdl.Desktop.Platform.Styles;component/buttonresources.xaml" />
+			</ResourceDictionary.MergedDictionaries>
+			<converters:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
+			<converters2:BooleanToVisibilityConditionalConverter x:Key="ConditionalConverter"/>
+			<ObjectDataProvider x:Key="ReplaceEnum"
+			                    MethodName="GetValues" ObjectType="{x:Type System:Enum}">
+				<ObjectDataProvider.MethodParameters>
+					<x:Type TypeName="enums:ReplacementEnum"/>
+				</ObjectDataProvider.MethodParameters>
+			</ObjectDataProvider>
+			<ObjectDataProvider x:Key="TargetEnum"
+			                    MethodName="GetValues" ObjectType="{x:Type System:Enum}">
+				<ObjectDataProvider.MethodParameters>
+					<x:Type TypeName="enums:TargetLanguageEnum"/>
+				</ObjectDataProvider.MethodParameters>
+			</ObjectDataProvider>
+		</ResourceDictionary>
+	</UserControl.Resources>
+	<GroupBox Style="{StaticResource Sdl.GroupBox.SingleLineStyle}"  Margin="5,0,0,0" Header="Settings">
+		<Grid>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto"/>
+			<RowDefinition Height="Auto"/>
+			<RowDefinition Height="Auto"/>
+			<RowDefinition Height="Auto"/>
+			<RowDefinition Height="Auto"/>
+			<RowDefinition Height="Auto"/>
+		</Grid.RowDefinitions>
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition Width="Auto"/>
+			<ColumnDefinition Width="Auto"/>
+		</Grid.ColumnDefinitions>
+		<CheckBox Grid.Row="0" Grid.Column="0"
+				  Width="Auto"
+				  Margin="5 10 5 25"
+		          Style="{StaticResource Sdl.Checkbox.GenericStyle}"
+		          Content="Overwrite Target Files"
+		          IsChecked="{Binding OverwriteTargetFiles, UpdateSourceTrigger=PropertyChanged}"/>
+		<ComboBox Grid.Row="1" Grid.Column="0"
+				  Margin="5 10 15 25"
+				  HorizontalContentAlignment="Center"
+				  Width="140"
+		          Style="{StaticResource Sdl.ComboBox.Generic.Style}"
+				  ItemsSource="{Binding Source={StaticResource ReplaceEnum}}"
+				  SelectedIndex="1"
+				  SelectionChanged="Selector_OnSelectionChanged"/>
```

Comment:
Please remove this action from the View and handle it into view model using Selected Item.